docker-init-influxdb.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+set -e
+
+AUTH_ENABLED="$INFLUXDB_HTTP_AUTH_ENABLED"
+
+if [ -z "$AUTH_ENABLED" ]; then
+	AUTH_ENABLED="$(grep -iE '^\s*auth-enabled\s*=\s*true' /etc/influxdb/influxdb.conf | grep -io 'true' | cat)"
+else
+	AUTH_ENABLED="$(echo ""$INFLUXDB_HTTP_AUTH_ENABLED"" | grep -io 'true' | cat)"
+fi
+
+INIT_USERS=$([ ! -z "$AUTH_ENABLED" ] && [ ! -z "$INFLUXDB_ADMIN_USER" ] && echo 1 || echo)
```

Comment:
Is there a reason to use `! -z` instead of `-n`?

And are the environment variables in this file documented anywhere?

Reply:
Yea, they're documented in the docker docs themselves. I copied the file directly from there.

I'm not sure at the moment how we should keep the two connected with each other. I just thought this feature was cool and a decent way to allow someone to more easily build a Docker image from master.